source/_docs/guides/drupal8-commandline.md

```diff
@@ -0,0 +1,376 @@
+
+If you have trouble with these steps, [see our full installation documentation](https://pantheon.io/docs/terminus/install/).
+
+## Create your site with Dev, Test, and Live environments
+
+#### See your options for creating new sites
+
+Pantheon offers a few choices for spinning up new sites.
+You can see them all with.
+
+`terminus upstream:list`
+
+The one we want is `Drupal 8`.
+
+`terminus upstream:list | grep "Drupal 8"`
+
+That command shows us the unique ID for our Drupal 8 source repository.
+We will copy that ID (8a129104-9d37-4082-aaf8-e6f31154644e) into the next command for creating a new site.
+
+#### Creating the Drupal 8 site.
+
+In this command you will need to pick both a machine name and a label for your site.
+In my case, I'm picking `steve-site-d8` as my site's machine name and "Steve's Site D8" as the label.
+Finally, I'm adding that unique ID for the Drupal 8 source.
+
+So the command I am running is:
+
+`terminus site:create steve-site-d8 "My Site D8" 8a129104-9d37-4082-aaf8-e6f31154644e`
+
+If you are copy and pasting these examples, you will need to replace `steve-site-d8` in each one.
+
+There's an additional option you can put on this command for organization ID `--org`.
+If you are walking through this guide as part of an in person training or if you work as part of a team that uses Pantheon, you might want to associate this site with your organization.
+To get your organization's ID, run `terminus org:list`.
+So your command might look like
+
+`terminus site:create steve-site-d8 "My Site D8" 8a129104-9d37-4082-aaf8-e6f31154644e --org=123456-abcd-1234--abcd-1234567890`
+
+#### Getting a link to your site's dashboard
+
+Even though we are using the command line as much as possible, you may find it helpful to open the Pantheon Dashboard in your browser and leave it open as you walk through this guide.
+The dashboard will respond as you run later commands for operations like deploying code.
+
+`terminus dashboard:view steve-site-d8  --print`
+
+#### Connection information
+
+In your browser dashboard you can easily get connection information.
+
+![Git and MySQL connection information from the Pantheon dashboard](/source/docs/assets/images/drupal8-commandline--connection-info.png)
+
+That same information is available in Terminus.
+Run this command replacing `steve-site-d8` with the name of the site you selected.
+
+`terminus connection:info steve-site-d8.dev`
+
+That command will show you some of the connection info.
+If you want to see all of the options available to you, run
+
+`terminus connection:info --help`
+
+Adding `--help` to any command will give you information about the options and parameters the command can use.
+In this case the `help` lets us know that we can specify different fields.
+So we can use the `--fields` to say that we want just the MySQL connection information.
+
+`terminus connection:info steve-site-d8.dev --fields=mysql_username,mysql_host,mysql_password,mysql_url,mysql_port,mysql_database`
```

Comment:
You can also use ``terminus connection:info steve-site-d8.dev --fields="mysql*"` here.